test/exercism/html_formatter_test.rb

```diff
@@ -37,11 +48,11 @@ def get_lexer(language)
   end
 
   def load_sample(name, language)
-    code = load_example name
-    @raw_lines = code.lines
+    @code = load_example name
+    @raw_lines = @code.lines
     lexer = get_lexer language
 
-    @lexemes = lexer.lex(code)
+    @lexemes = lexer.lex(@code)
     @output = @formatter.format(@lexemes)
 
     @doc = Loofah::HTML::DocumentFragment.parse(@output)
```

Comment:
All of these aid in debugging. I don't have strong opinions on these, but test code is not public API, and such constructs are acceptable IMO. Will change.